lib/pyfrc/mains/cli_sim.py

```diff
@@ -15,47 +15,6 @@ class PyFrcSim:
 
     def __init__(self, parser):
         pass
-    
-    def _load_config(self, config_file):
```

Comment:
Why did you get rid of this? It sets up defaults in a way that makes them easy to find -- and because there are defaults, it reduces the amount of conditional logic needed to access it later.